src/python/kalkulator/README.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Lærerveiledning - Kalkulator
+level: 3
+language: nb
+---
+
+
+# Om oppgaven {.activity}
+
+I denne oppgaven skal elevene lage en kalkulator helt på egenhånd. Det er meningen
+at kalkulatoren skal kunne legge sammen, trekke fra, gange og dele to tall på hverandre.
+
+
+## Oppgaven passer til: {.check}
+
+ __Fag__: Matematikk, Programmering 
+
+ __Anbefalte trinn__:7.-10. trinn
+
+ __Tema__: Aritmetikk, if-setninger, brukerinteraksjon, funksjoner
+
+ __Tidsbruk__: Dobbeltime
+
+
+## Kompetansemål {.challenge}
+
+ - [ ] __Matematikk, 4. trinn__: bruke matematiske symboler og uttrykksmåter for å uttrykke matematiske sammenhenger i oppgaveløsning
+
+ - [ ] __Programmering, 10. trinn__: bruke flere programmeringsspråk der minst ett er tekstbasert
+
+ - [ ] __Programmering, 10. trinn__: bruke grunnleggende prinsipper i programmering, slik som løkker, tester, variabler, funksjoner og enkel brukerinteraksjon
+
+ - [ ] __Programmering, 10. trinn__: omgjøre problemer til konkrete delproblemer, vurdere hvilke delproblemer som lar seg løse digitalt, og utforme løsninger for disse
+
+
+## Forslag til læringsmål {.challenge}
+
+ - [ ] Elevene klarer å dele opp programmet sitt i logiske funksjoner
+
+ - [ ] Elevene klarer å skrive selvstendig kode, basert på små hint
+
+ - [ ] Elevene er i stand til å bruke matematiske symboler til å kode en enkel kalkulator
+
+
+## Forslag til vurderingskriterier {.challenge}
+
+- [ ] Eleven viser middels måloppnåelse ved å fullføre oppgaven.
+
+- [ ] Eleven viser høy måloppnåelse ved å videreutvikle egen kode basert på oppgaven, for eksempel ved å gjøre en eller flere av variasjonene nedenfor.
+
+ 
+## Forutsetninger og utstyr {.challenge}
+
+ - [ ]  __Forutsetninger__: Kjennskap til while-løkker, if-setninger, funksjoner. 
+
+ - [ ]  __Utstyr__: Datamaskin med Python installert.
+
+
+# Fremgangsmåte
+
+ Her kommer tips, erfaring og utfordringer til de ulike stegene i den faktiske oppgaven. [Klikk her for å se oppgaveteksten.](../kalkulator/kalkulator.html){target=_blank}
```

Comment:
denne setningen står i malen, så isåfall burde alle lærerveiledninger som er skrevet endre denne teksten. da er det kanskje bedre om vi lar det være, også tar vi en runde og endrer alle i en PR senere.